react-time-track/src/views/edit-user.js

```diff
@@ -0,0 +1,62 @@
+/** @jsx jsx */
+import React from "react";
+import {jsx} from "@emotion/core";
+import {navigate} from "@reach/router";
+import {Li, Title, UserCard, Card, Button} from "../components/ui";
+import {getUser, editUser} from "../services/user";
+
+function EditUser({user_id}) {
+    const [user, setUser] = React.useState({
+			name: "",
+			email: "",
+			role: "",
+			rate: 0
+		});
+
+    React.useEffect(()=> {
+      getUser(user_id)
+      .then(data => {setUser(data)
+    	});
+		}, []);
+
+	function handleChange(e, key) {
+		setUser({...user, [key]: e.target.value})
+	}
+
+	function handleSubmit(e) {
+		e.preventDefault();
+		editUser(user_id, user)
+		.then(response => {
+			navigate("/users");
+		})
+		.catch(response => {
+			""
+		});
+	}
+
+	return (
+		<div 
+			css={{
+			maxWidth: "500px",
+			margin: "auto",
+			display: "flex",
+			flexDirection: "column",
+			alignItems: "center"
+			}}
+		>
+
+			<form css={{marginTop: "5em", display: "flex", flexDirection: "column", width: "50%", fontSize: "20px"}} onSubmit={handleSubmit}>
+				Name: <input type="text" name="user-name" onChange={(event)=>handleChange(event, "name")} value={user.name} css={{fontSize: "16px", color: "gray", marginBottom: "10px"}}/>
+				Email: <input type="text" name="user-email" onChange={(event)=>handleChange(event, "email")} value={user.email} css={{fontSize: "16px", color: "gray", marginBottom: "10px"}}/>
+				Role: <input type="text" name="user-role" onChange={(event)=>handleChange(event, "role")} value={user.role} css={{fontSize: "16px", color: "gray", marginBottom: "10px"}}/>
+				Rate: <input type="text" name="user-rate" onChange={(event)=>handleChange(event, "rate")} value={user.rate} css={{fontSize: "16px", color: "gray", marginBottom: "10px"}}/>				
```

Comment:
Hi, what do you think about refactoring the object within css because it is repeated in all the input elements?, You can create a const variable with that value and only call it in each element

Reply:
```
const inputStyles = {
  fontSize: "16px",
  color: "gray",
  marginBottom: "10px"
}
```
and 
```
css= {inputStyles}
```
May be we should set a daily checklist, just to improve our job as software engineers. =|
I think I read something related. I'll look after it later.

Reply:
I think your point of view is correct. I will refactor those.

Reply:
thanks @test0n3.